inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -1352,7 +1352,6 @@ bool MKLDNNMVNNode::checkAxesSuitability(const std::shared_ptr<const ngraph::Nod
     if (mvn != nullptr && node->get_input_size() == 2) {
         if (auto axesNode = dynamic_cast<ngraph::op::v0::Constant*>(mvn->get_input_node_ptr(1))) {
             auto axesVal = axesNode->cast_vector<int>();
-            std::sort(axesVal.begin(), axesVal.end());
```

Comment:
This change is needed in 2021.3

Reply:
@mvafin Could you create a PR which will revert all redundant changes for 2021.3 release? Because the original PR contains both changes.
If you create a PR, I will close this one.